### PR TITLE
feat(arch): add Q-ReAlign Qwen3.5-VL quality metric

### PR DESCRIPTION
**Pull request by Frank-Y-Zheng**
# PR Title

feat(arch): add Q-ReAlign Qwen3.5-VL quality metric

# PR Body

## Summary

- Add `qrealign`, a Q-ReAlign visual-quality scorer built on Qwen3.5-VL, with scores collapsed from `excellent/good/fair/poor/bad` logits into `[0, 1]`.
- Register `qrealign`, `qrealign-mini`, `qrealign-lite`, and `qrealign-pro` as NR metrics with Mini/Lite/Pro Hugging Face checkpoints.
- Add vendored Qwen3.5 compatibility support for `transformers` 5.0/5.1, while using native `qwen3_5` support on newer releases.
- Update the README and model card, bump `transformers>=5.0`, and fix legacy `qalign` level-token lookup for newer `LlamaTokenizer` behavior.

## Validation

- `python -m py_compile pyiqa/archs/qrealign_arch.py pyiqa/archs/qrealign/qrealign_compat.py pyiqa/archs/qrealign/qwen3_5_src/configuration_qwen3_5.py pyiqa/archs/qrealign/qwen3_5_src/tokenization_qwen3_5.py`
- `python -c "import ast; m=ast.parse(open('pyiqa/default_model_configs.py').read()); print('default_model_configs ast ok')"`
- `python -c "import ast; [ast.parse(open(p).read()) for p in ['pyiqa/archs/qrealign_arch.py','pyiqa/archs/qrealign/qrealign_compat.py','pyiqa/archs/q_align/modeling_mplug_owl2.py']]; print('ast ok')"`

Full package import and runtime model checks were not run in this local environment because `torch` is not installed. The commit message records prior validation on `transformers` 5.0 and 5.2: all Q-ReAlign variants load and score, results are bit-identical between those versions, and legacy `qalign` / FR calibration are unchanged.

# Submit Commands

```bash
gh auth login -h github.com
git push -u origin feat/qrealign-metric
gh pr create \
  --base main \
  --head feat/qrealign-metric \
  --title "feat(arch): add Q-ReAlign Qwen3.5-VL quality metric" \
  --body-file PR_qrealign.md
```

### CODE REVIEW

**Comment by chaofengc** [comment]
Great work! Thank you so much for your valuable contribution. 🙏
